### PR TITLE
Wording changes in openEQUELLA section

### PR DESCRIPTION
**Pull request by goetzk**
Removed the placeholders which don't seem useful any longer and updated some
other wording in the install steps.

### CODE REVIEW

File: guides/InstallationAdminGuide.md [hunk 1]
```diff
@@ -227,21 +227,19 @@ openEQUELLA provides files for installation that can be used on all platforms. T
...
-
 ### Begin openEQUELLA Installation 
 
 This procedure describes installing openEQUELLA using a graphical interface. The examples shown are using Windows although other GUIs such as X-windows will be similar. The wizard pages provide information on the required details. Read each page before entering information.
 
-1. PLACEHOLDER FOR WHERE THE INDIVIDUAL CAN DOWNLOAD THE equella-6.4-GAx Installer file.
-2. Extract the equella-installer-xxxx.zip file to a temporary directory.
+Note: It may be possible to install without requiring a GUI; see https://groups.google.com/a/apereo.org/d/msg/equella-users/v-MMOuoa7mk/UDbmSkKRAwAJ for discussion if this is important to you.
+
+1. Download equella-installer-x.x.zip (substitute x for the current minor version) from https://github.com/equella/Equella/releases
```
Comment (cbeach47) [inline guides/InstallationAdminGuide.md]: I would change it to:

Obtain an openEQUELLA installer.  You can either download a demo installer from https://github.com/equella/Equella/releases or build a production installer via `sbt`.  Demo installers do not contain Kaltura integration or Oracle DB drivers, and use a self-signed Java signing certificate.  The file format is equella-installer-x.y.zip (substitute x for the current major version, and y for the current minor version)

File: guides/InstallationAdminGuide.md [hunk 2]
```diff
@@ -227,21 +227,19 @@ openEQUELLA provides files for installation that can be used on all platforms. T
...
+2. Extract the equella-installer-x.x.zip file to a temporary directory.
```
Comment (cbeach47) [inline guides/InstallationAdminGuide.md:237]: Change to:
Extract the equella-installer-x.y.zip file to a temporary directory.

File: guides/InstallationAdminGuide.md [hunk 3]
```diff
@@ -227,21 +227,19 @@ openEQUELLA provides files for installation that can be used on all platforms. T
...
-
-1.  ... In Progress...  Need to insert steps required to get install files 
-
 ### Begin openEQUELLA Installation 
 
 This procedure describes installing openEQUELLA using a graphical interface. The examples shown are using Windows although other GUIs such as X-windows will be similar. The wizard pages provide information on the required details. Read each page before entering information.
 
-1. PLACEHOLDER FOR WHERE THE INDIVIDUAL CAN DOWNLOAD THE equella-6.4-GAx Installer file.
-2. Extract the equella-installer-xxxx.zip file to a temporary directory.
+Note: It may be possible to install without requiring a GUI; see https://groups.google.com/a/apereo.org/d/msg/equella-users/v-MMOuoa7mk/UDbmSkKRAwAJ for discussion if this is important to you.
```
Comment (cbeach47) [inline guides/InstallationAdminGuide.md]: We can use stronger language, such as:

Note: It is possible to install without requiring a GUI.  While not widespread as a production installation method, it's consistently used in the docker logic. see ....

**Review by cbeach47 — CHANGES_REQUESTED** [review]
Good catch - the way to find a release was a WIP during open sourcing of oEQ, and this page was a loose end.

I mentioned a couple of changes I'd suggest.

**Comment by goetzk** [comment]
Hi @cbeach47 ,
I've brought those suggestions in verbatim in the latest commit.

**Review by cbeach47 — APPROVED** [review]

**Pull request merged**